app/src/main/java/net/gsantner/markor/activity/NoteActivity.java

```diff
@@ -126,6 +127,10 @@ public void onFocusChange(View v, boolean hasFocus) {
                 }
             }
         });
+
+        _extensions = new String[]{ Constants.MD_EXT1, Constants.MD_EXT2, Constants.MD_EXT3,Constants.MD_EXT4,
```

Comment:
could be pur in constants too, for reuse

Reply:
Didn't get you. "Could you pur"? 
Rest you can tell me which all extensions you want and I will add them  or you can add it on your own?

Reply:
put

everything markdown like + txt

Reply:
to move the array to Constants and make it final static